code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -278,7 +278,8 @@
 		/obj/item/borg/cyborghug/medical,
 		/obj/item/stack/medical/gauze/cyborg,
 		/obj/item/organ_storage,
-		/obj/item/borg/lollipop)
+		/obj/item/borg/lollipop,
+		/obj/item/stack/nanopaste)

```

Comment:
Ehhhh do we want mediborgs having access from round start? I'm not so sure.

Reply:
Regardless of that, cyborgs can't just get a regular item put into them like that. It needs to be made into a cyborg-specific subtype first; look at how other medical stacks are handled.

Reply:
TIL how borg modules worked.

Reply:
Can mediborgs now nanopaste themselves? 🤔